src/components/Modal/Modal.tsx

```diff
@@ -74,10 +74,14 @@ export function Modal(props: ModalProps) {
 
   const [hasScroll, setHasScroll] = useState(forceScrolling ?? false);
 
-  useResizeObserver(modalBodyRef, ({ target }) => {
-    if (forceScrolling === undefined && !isFixedHeight) {
-      setHasScroll(target.scrollHeight > target.clientHeight);
-    }
+  useResizeObserver({
```

Comment:
@bdow sanity check on this in particular, I moved over to `react-aria`'s `useResizeObserver`.

Reply:
Nice! Interesting that React-Aria provides this util/hook.